TypeCobol/Compiler/Sql/Model/SelectClause.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;
```

Comment:
In this file:
- for StarSelection: add a Dump override that write nothing (since the class does not contain anything meaningful)
- for TableOrViewAllColumnsSelection: add a Dump method that write the TableOrViewOrCorrelationName
  - for TableOrViewOrCorrelationName, check what ToString() of SymbolReference return. if it's ok use it otherwise write specific code to display the name